e2e/specs/settings/keyboard_shortcuts.test.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+'use strict';
+
+const fs = require('fs');
+
+const robot = require('robotjs');
+
+const {SHOW_SETTINGS_WINDOW} = require('../../../src/common/communication');
+
+const env = require('../../modules/environment');
+const {asyncSleep} = require('../../modules/utils');
+
+describe('settings/keyboard_shortcuts', function desc() {
+    this.timeout(30000);
```

Comment:
I guess this is because I don't know the test framework you're using, but it seems odd that we'd be setting a manual timeout here and it wouldn't be a default thing.

Also, eww. Binding `this` to something random 😛